packages/atomic/src/components/facets-v1/atomic-rating-range-facet/atomic-rating-range-facet.tsx

```diff
@@ -0,0 +1,252 @@
+      currentValues.push(
+        buildNumericRange({
+          start: Math.round((i + 1) * this.scaleFactor * 100) / 100,
+          end: Math.round(this.maxValueInIndex * 100) / 100,
+          endInclusive: true,
+        })
+      );
+    }
+    return currentValues;
+  }
+
+  private formatFacetValue(facetValue: NumericFacetValue) {
+    return this.bindings.i18n.t('to', {
+      start: facetValue.start,
+      end: facetValue.end,
+    });
+  }
+
+  private renderHeader() {
+    return (
+      <FacetHeader
+        i18n={this.bindings.i18n}
+        label={this.label}
+        onClearFilters={() => this.facet.deselectAll()}
+        numberOfSelectedValues={this.numberOfSelectedValues}
+        isCollapsed={this.isCollapsed}
+        onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+      ></FacetHeader>
+    );
+  }
+
+  private renderLabelText(facetValue: NumericFacetValue) {
+    return (
+      <span
+        part="value-label"
+        class="ml-1 flex items-center truncate group-hover:underline group-hover:text-primary"
+        style={
+          facetValue.state === 'selected'
+            ? {['font-weight']: 'bold'}
+            : {['font-weight']: 'normal'}
+        }
+      >
+        {facetValue.start === this.maxValueInIndex ? (
+          <span>{this.bindings.i18n.t('only')}</span>
+        ) : (
+          this.bindings.i18n.t('& up')
+        )}
+      </span>
+    );
+  }
+
+  private renderValue(facetValue: NumericFacetValue, onClick: () => void) {
+    const displayValue = this.formatFacetValue(facetValue);
+    const isSelected = facetValue.state === 'selected';
+    return (
+      <FacetValueLink
+        displayValue={displayValue}
+        numberOfResults={facetValue.numberOfResults}
+        isSelected={isSelected}
+        i18n={this.bindings.i18n}
+        onClick={onClick}
+      >
+        <FacetValueIconRating
+          numberOfTotalIcons={this.maxValueInIndex}
+          numberOfActiveIcons={facetValue.start}
+          icon={this.icon}
+        ></FacetValueIconRating>
+        {this.renderLabelText(facetValue)}
+      </FacetValueLink>
+    );
+  }
+
+  private renderValuesContainer(children: VNode[]) {
+    return (
+      <ul part="values" class="mt-3">
+        {children}
+      </ul>
+    );
+  }
+
+  private renderValues() {
+    return this.renderValuesContainer(
+      this.valuesToRender.map((value) =>
+        this.renderValue(value, () => this.facet.toggleSingleSelect(value))
+      )
+    );
+  }
+
+  private get valuesToRender() {
+    return this.facetState.values.filter(
+      (value) => value.numberOfResults || value.state !== 'idle'
+    );
+  }
+
+  public render() {
+    if (this.searchStatusState.hasError) {
+      return;
+    }
+
+    if (!this.searchStatusState.firstSearchExecuted) {
+      return (
+        <FacetPlaceholder
+          numberOfValues={this.numberOfIntervals}
+        ></FacetPlaceholder>
+      );
+    }
+
+    if (!this.facetState.values.length) {
```

Comment:
Here and for the other rating facet, the condition is a bit more complex, we need to filter or values that either have a positive number of values or selected values. You an look at other v1 facets for reference 
<img width="1062" alt="Screen Shot 2021-07-26 at 11 54 37 AM" src="https://user-images.githubusercontent.com/4923043/127020373-8eecd795-43c0-48b5-81de-71cf08ebc09c.png">

Reply:
I can't reproduce this on my side, I get:
![Screen Shot 2021-07-26 at 1 31 03 PM](https://user-images.githubusercontent.com/83350303/127032956-9980d1d8-1287-4f05-a04c-dde1d5dcb672.png)